nutils/types.py

```diff
@@ -1484,6 +1484,55 @@ def del_lru(self):
       lru = self._sorted_keys.pop()
     del self[lru]
 
+def lru_cache(func=None, maxsize=None):
+  '''Buffer-aware LRU cache.
+
+  Return values are cached for immutable arguments are cached, otherwise
```

Comment:
Typo: Return values -are cached- for immutable arguments are cached, ...

Reply:
fixed